src/styles/utilities/spacing.js

```diff
@@ -353,4 +353,8 @@ export default {
     pb10Percentage: {
         paddingBottom: '10%',
     },
+
+    gap1: {
+        gap: 4,
+    },
```

Comment:
Seems like Android native ignores the style "gap"

https://github.com/Expensify/App/issues/16864

Reply:
Shoot, that's not ideal. Maybe we should have a lint rule that prohibits `gap` css prop in that case?

Reply:
[Looks like it is supported in 0.71](https://github.com/styled-components/styled-components/issues/3628#issuecomment-1397092044), I believe we are working on updating to that version now.